bcda/cclf/cclf_test.go

```diff
@@ -245,8 +259,12 @@ func (s *CCLFTestSuite) TestImportCCLF9() {
 	db := database.GetGORMDbConnection()
 	defer database.Close(db)
 
-	db.Unscoped().Delete(&models.CCLFBeneficiaryXref{})
-	db.Unscoped().Delete(&models.CCLFFile{})
+	var existngCCLFFiles []models.CCLFFile
+	db.Where("aco_cms_id = ?", "A0001").Find(&existngCCLFFiles)
+	for _, cclfFile := range existngCCLFFiles {
+		err := cclfFile.Delete()
+		assert.Nil(err)
+	}
```

Comment:
Can we make a cleanup function in the test file that does this work, maybe accepting the ACO ID as a parameter?

Reply:
Sure.  Can we include that as part of the test cleanup epic?  I only did this so I could get through my tests and I don't want to get to wide ranging with this PR.